### ISSUE TITLE
Action Items in iTunes not accessible

### ISSUE THREAD

**Issue by bobdavcav** [nvaccess/nvda#15653]
### Steps to reproduce:
1. Preview a few song using iTunes. This can be done on your phone or PC, doesn't matter. 
2. On your PC, open iTunes, navigate to the store menu, and choose Wish List. 
3. Once that page opens, use heading navigation to jump down to preview history. 
4. Tab through this page. You should see a group of items associated with each song. These should be a preview button, a link presumably to take you to the album on which the song is on, and a combo box with more actions. 
5. Notice that when the combo box is encountered and you have NVDA set to automatically go into focus mode when a form field is encountered, that it does in fact do so. 
### Actual behavior:

When pressing down arrow from step 5 above, NVDA leaves focus mode and the combo box in question and continues navigating the page as if it was never in focus mode. 
### Expected behavior:

I expect to be able to interact with the combo box as I would with any other combo box. 
#### NVDA installed/portable/running from source:
It doesn't matter whether running NVDA from a portable copy or it's installed on the system. 
#### NVDA version:
It first appears in NVDA 2023.2, continues in NVDA 2023.3 Beta 4. 
#### Windows version:
Edition	Windows 10 Home
Version	22H2
OS build	19045.3570
Experience	Windows Feature Experience Pack 1000.19052.1000.0

#### Name and version of other software in use when reproducing the isuld e:
I would provide the exact iTunes version, but NVDA does not read the About iTunes window. 
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
The last version to behave normally was 2023.1. 
#### If NVDA add-ons are disabled, is your problem still occurring?
I haven't tried, but would assume so. 
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
I don't think that would make a difference, but I haven't tried.

**Comment by seanbudd** [comment]
Welcome @bobdavcav ,

We are unable to process this issue as it stands.

- Please complete the required steps for reporting an issue. Please test with add-ons disabled. Please also run the COM registration fixing tool.
- Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences. Press `NVDA+f1` when focused on the combo box control. This will log developer info for the control.
- This is likely a problem with the implementation of iTunes. Have you tested with 2023.1 recently? Are you certain iTunes has not updated in this time?

Kind Regards,
NV Access Software Developers

**Comment by bobdavcav** [comment]
I cannot remember whether iTunes has updated or not since the last
NVDA update. What I do know though is that this was an issue I
encountered as soon as I updated to NVDA 2023.2, but on trying 2023.1
it worked as expected. Running with addons disabled isn't going to do
any good, as I get the same result with my installed copy of NVDA
2023.2 running only Numpad Nav Mode as I get with my portable copy of
NVDA 2023.3 beta 4 with no addons.

**Comment by seanbudd** [comment]
> as I get with my portable copy of NVDA 2023.3 beta 4 with no addons.

Does that mean you have test with add-ons disabled? If so, I will update the issue template to "Yes".

>but on trying 2023.1 it worked as expected. 

Does that mean you have tested again with 2023.1 after you installed 2023.2?

When possible, can you please address these extra items

>* Please complete the required steps for reporting an issue. Please test with add-ons disabled. Please also run the COM registration fixing tool.
>* Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences. Press NVDA+f1 when focused on the combo box control. This will log developer info for the control.

**Comment by bobdavcav** [comment]
I'm assuming running a portable copy of NVDA without addons is
functionally the same as running NVDA with addons disabled, correct?
As I said, my portable copy of 2023.3 beta 4 does not have any addons
installed. Yes, I have done this with NVDA 2023.1 after installing
2023.2. For more information about the timeline here, see this thread
I posted to the users list:
https://nvda.groups.io/g/nvda/topic/101331514#111167

**Comment by seanbudd** [comment]
@bobdavcav - can you please complete the additional steps outlined in https://github.com/nvaccess/nvda/issues/15653#issuecomment-1780437976 ?

I'm unable to reproduce this issue as iTunes won't let me open the wishlist, throwing a consistent error.
Are there other controls that inaccessible that I can test against?

**Comment by bobdavcav** [comment]
Sorry, just now picking this up, as my Github account isn't linked to
my personal email. I did not get a clear answer to my question,
running a copy of NVDA without addons is functionally the same as
running one with addons disabled, right? If so, running with addons
disabled isn't going to do anything, as the beta version I have does
not have any adons installed in the first place. I can attempt to get
a log for you tomorrow. As for answering your question, while there
are other issues with iTunes, they don't relate to this particular
issue. For years, I don't think any screen reader has read the names
of the menus in the menu bar, and NVDA does not read the "about
iTunes" window. Can you get to the store at all? If so, try to take
any action on a song and you should see what I'm talking about.

**Comment by seanbudd** [comment]
> I did not get a clear answer to my question,
running a copy of NVDA without addons is functionally the same as
running one with addons disabled, right? 

Yep that's the case - so no need to retest.
I've reproduced the issue now so there is no need for a log.

I managed to confirm this issue by attempting to navigating to the actions for a song. The menu is unusable.
This is definitely a regression from 2023.1, which I tested and could read the action items correctly.
